File: go/remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
package main

import(
	"fmt"
)

func removeDuplicates(nums []int) int {
	index := 1
	for i := 1; i <len(nums); i++{
		if nums[i] != nums[i-1]{
			nums [index] = nums[i]
			index++
		}
	}


	return index
}

func main(){
	array1 := []int {1 , 1 , 2, 2, 3, 4, 4}
	fmt.Println("answer to the first example is:", removeDuplicates(array1))
}

File: js/remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
function removeDuplicates(nums) {
    let index = 1
    for (let i = 1; i < nums.length; i++) {
        if (nums[i] !== nums[i - 1]) {
            nums[index] = nums[i]
            index++
        }
    }
    return index
}

let array1 = [ 1 , 1 , 2, 2, 3, 4, 4 ]
	console.log("answer to the first example is:", removeDuplicates(array1))
